tests/Unit/Utilities/Test_Tuple.cpp

```diff
@@ -21,9 +21,35 @@ struct tuple_fold_plus {
   }
 };
 /// [tuple_fold_struct_defn]
+
+/// [tuple_transform_negate]
+struct negate {
+  template <typename T, typename Index, typename S>
+  void operator()(const T& element, Index /*index*/,
+                  S& second_tuple_element) const noexcept {
+    std::get<Index::value>(second_tuple_element) = -element;
+  }
+};
+/// [tuple_transform_negate]
+struct negate_if_sum_less {
+  template <typename T, typename Index, typename S>
+  void operator()(const T& element, Index /*index*/, S& second_tuple_element,
+                  const gsl::not_null<double*> state,
+                  const std::string& test_sentence,
+                  const std::string& test_sentence2) const noexcept(false) {
+    CHECK(test_sentence == "test sentence");
+    CHECK(test_sentence2 == "test sentence2");
+    if (*state < 8.0) {
+      *state += element;
+      std::get<Index::value>(second_tuple_element) = -element;
+    } else {
+      std::get<Index::value>(second_tuple_element) = element;
+    }
+  }
+};
 }  // namespace
 
-SPECTRE_TEST_CASE("Unit.Utilities.tuple_fold", "[Utilities][Unit]") {
+SPECTRE_TEST_CASE("Unit.Utilities.Tuple", "[Utilities][Unit]") {
```

Comment:
Instead of just indenting these and having one giant function, why not have separate functions in the anonymous namespace that are then called from a since test? This is the pattern we have adopted in most places